appium/webdriver/extensions/action_helpers.py

```diff
@@ -149,3 +149,87 @@ def flick(self, start_x, start_y, end_x, end_y):
             .release()
         action.perform()
         return self
+
+    def zoom_element(self, element):
+        """Zooms in on an element a certain amount
+
+        :Args:
+         - element - the element to zoom
+
+        :Usage:
+            driver.zoom(element)
+        """
+        multi_touch = MultiAction()
+        size=element.size
+        position = element.location
+        coordinate_x=position.get('x')
+        coordinate_y=position.get('y')
+        width=size.get('width')
+        height=size.get('height')
+        center = {'x': coordinate_x+width/2, 'y': coordinate_y+height/2}
+        action1 = TouchAction(self)
```

Comment:
Can you consider to use W3C actions instead of TouchAction?

Reply:
Thanks!  FYI: https://github.com/appium/java-client/commit/6b216815411a0d7ddf1679c1ceb0c10c5a779b3d#diff-f50beb490f1d493c61c0e8d7a963e5e8

Reply:
@KazuCocoa Can you please let me know why we should use W3C actions ?

Reply:
We have no exact plan when we drop MJSONWP so far, but we're going to focus on W3C protocol. Then, it will happen on the client side as well.

Reply:
ok! let me do some changes~